### DISCUSSION TITLE
Problem with price per hour

### DISCUSSION THREAD

**Discussion by Nadine9418** [kimai/kimai#4313] [Questions & Answers] [answered]
I have a problem with my price per hour. In my settings i have to choose my price/hour. But i have clients with different prices per hour. If i write my invoice the price isn't correct with the price per hour. The next problem is that i can't choose more than one price per hour.

![grafik](https://github.com/kimai/kimai/assets/145661401/81ed8776-ebaa-4519-b253-b09ab0e6fe73)

**Comment by kevinpapst** [comment, marked as answer]
Nope, the problem ist that you did not find the correct documentation yet:
https://www.kimai.org/documentation/rates.html

You can configure rates on many levels => you are looking for the customer detail page where you can add customer specific prices

**Comment by richard-lee-863** [comment]
Is it possible to have more than one rate per user for a project or customer though? It doesn't seem possible.

My use case is we have Office and Onsite work rates. Which will be different per user and per project. 

It seems I can set up two rates per Activity for those. But ideally Project A can have different activity rates from Project B.

**Reply by kevinpapst** [reply]
> Is it possible to have more than one rate per user for a project or customer though? It doesn't seem possible.

Yes through activities.

> But ideally Project A can have different activity rates from Project B.

Use project specific activities, not global ones.
